CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog
 
+## v0.19.0 (2018-2-15)
+
+### Enhancements
+
+#### Taking Screenshots of Individual Page Elements ([#1496](https://github.com/DevExpress/testcafe/issues/1496))
+
+We have added the [t.takeElementScreenshot](https://devexpress.github.io/testcafe/documentation/test-api/actions/take-screenshot.html#take-a-screenshot-of-a-page-element) action that allows you to take a screenshot of an individual page element.
```

Comment:
Should we mention 'screenshot path' option? This example will not work without it.